Rubberduck.RibbonDispatcher/AbstractRibbon.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using stdole;
+using Microsoft.Office.Core;
+
+using Rubberduck.RibbonDispatcher.Abstract;
+
+namespace Rubberduck.RibbonDispatcher
+{
+    using System.Collections.Generic;
+    using static RibbonControlSize;
+    using static System.Globalization.CultureInfo;
+
+    /// <summary>(All) the callbacks for the Fluent Ribbon.</summary>
+    /// <remarks>
+    /// The callback names are chosen to be identical to the corresponding xml tag in
+    /// the Ribbon schema, except for:
+    ///  - PascalCase instead of camelCase; and
+    ///  - In some instances, a disambiguating usage suffix such as OnActionToggle(,)
+    ///    instead of a plain OnAction(,).
+    /// </remarks>
+    [ComVisible(true)][CLSCompliant(true)]
+    public abstract class AbstractRibbon {        
+        public virtual void OnRibbonLoad(IRibbonUI ribbonUI) {
+            RibbonFactory         = new RibbonFactory(ribbonUI);
+        }
+        protected RibbonFactory     RibbonFactory    { get; private set; }    // A property because might be exposed in future
```

Comment:
misaligned to the other declarations. Comment at the end of the line is hard to read

Reply:
Also ... YAGNI. Don't prepare for the "might" but work with the "is"